### PR TITLE
Add cli tool.

### PR DESCRIPTION
**Pull request by Kineolyan**
Because I often want the info provided by _universions_ in my CLI, I am trying to create a CLI tool packed with this module.

The usage would be simple:
`universions java` -> "1.8.201"
`universions node` -> "12.2.5"

Closes: #22

### CODE REVIEW

**Comment by fabiencelier** [comment]
You should use `argparse` !
It is included in the core python module and does a lot of things nicely when building a CLI

File: cli.py [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+    """
+    Gets the version of this exact module.
+    From the advised alternatives in
+    https://packaging.python.org/guides/single-sourcing-package-version/, this reads from setup.py.
+    But there is a little problem: for a version such as 0.1.0-dev21, the attribute version returns
+    '0.1.0.dev21'. This issue is that the '-' is replaced by a '.', which makes it an invalid semver
+    """
+    raw_version = pkg_resources.require("universions")[0].version
+    version = raw_version.replace(".dev", "-dev")
+    return universions.parse_semver(version)
```
Comment (fabiencelier) [inline cli.py]: In latest version you can use `universions.__version__`

I think python requires the libraries version to be `.devN`, which is bad for SemVer. I need to investigate more.

Comment (Kineolyan) [reply cli.py]: Unfortunately not working with python 3.7.4 😞

**Comment by Kineolyan** [comment]
The code is now sync with the master branch and passing quality gates.

**Pull request merged**